src/list.scss

```diff
@@ -101,6 +101,107 @@ $block: #{$fd-namespace}-list;
         background: var(--sapList_Background);
       }
     }
+
+    &--action {
+      display: -webkit-box;
+      display: flex;
+      size: 2.5rem;
+      padding: 0 0.25rem;
+      -webkit-box-pack: center;
+      justify-content: center;
+      cursor: pointer;
+      border: var(--sapPageFooter_BorderColor);
+      background: var(--sapPageFooter_Background);
+      height: 3rem;
+
+      @include fd-ellipsis();
```

Comment:
Please add this include at the top of the block with the other includes

Reply:
Done